node/pkg/p2p/cutover.go

```diff
@@ -1,112 +1,19 @@
-		go func() {
-			time.Sleep(delay)
-			logger.Info("time to cut over to new quic-v1", zap.String("cutOverTime", cutOverTimeStr), zap.String("component", "p2pco"))
-			panic("p2pco: time to cut over to new quic-v1")
-		}()
-	}
-
-	return nil
-}
-
-// evaluateCutOverImpl performs the actual cut over check. It is a separate function for testing purposes.
-func evaluateCutOverImpl(logger *zap.Logger, cutOverTimeStr string, now time.Time) (bool, time.Duration, error) {
-	if cutOverTimeStr == "" {
-		return false, 0, nil
-	}
-
-	cutOverTime, err := time.Parse(cutOverFmtStr, cutOverTimeStr)
-	if err != nil {
-		return false, 0, fmt.Errorf(`failed to parse cut over time: %w`, err)
-	}
-
-	if cutOverTime.Before(now) {
-		logger.Info("cut over time has passed, should use new quic-v1", zap.String("cutOverTime", cutOverTime.Format(cutOverFmtStr)), zap.String("now", now.Format(cutOverFmtStr)), zap.String("component", "p2pco"))
-		return true, 0, nil
-	}
-
-	// If we get here, we need to wait for the cutover and then force a restart.
-	delay := cutOverTime.Sub(now)
-	logger.Info("still waiting for cut over time",
-		zap.Stringer("cutOverTime", cutOverTime),
-		zap.String("now", now.Format(cutOverFmtStr)),
-		zap.Stringer("delay", delay),
-		zap.String("component", "p2pco"))
-
-	return false, delay, nil
-}
-
-// getCutOverTimeStr returns the cut over time string based on the network ID passed in.
-func getCutOverTimeStr(networkID string) string {
-	if strings.Contains(networkID, "/mainnet/") {
-		return mainnetCutOverTimeStr
-	}
-	if strings.Contains(networkID, "/testnet/") {
-		return testnetCutOverTimeStr
-	}
-	return devnetCutOverTimeStr
-}
-
-// cutOverBootstrapPeers checks to see if we are supposed to cut over, and if so updates the bootstrap peers. It assumes that the string has previously been validated.
+// cutOverBootstrapPeers updates the bootstrap peers to reflect the new quic-v1. It assumes that the string has previously been validated.
 func cutOverBootstrapPeers(bootstrapPeers string) string {
-	if shouldCutOver() {
-		bootstrapPeers = strings.ReplaceAll(bootstrapPeers, "/quic/", "/quic-v1/")
-	}
-
-	return bootstrapPeers
+	return strings.ReplaceAll(bootstrapPeers, "/quic/", "/quic-v1/")
 }
 
-// cutOverAddressPattern checks to see if we are supposed to cut over, and if so updates the address patterns. It assumes that the string is valid.
+// cutOverAddressPattern updates the address patterns. It assumes that the string is valid.
 func cutOverAddressPattern(pattern string) string {
-	if shouldCutOver() {
-		if !strings.Contains(pattern, "/quic-v1") {
-			// These patterns are hardcoded so we are not worried about invalid values.
-			pattern = strings.ReplaceAll(pattern, "/quic", "/quic-v1")
-		}
+	if !strings.Contains(pattern, "/quic-v1") {
+		// These patterns are hardcoded so we are not worried about invalid values.
+		pattern = strings.ReplaceAll(pattern, "/quic", "/quic-v1")
```

Comment:
Seems like we should error out now that the cut is official or at a minimum log that the operator has an invalid configuration. What do you think?

Reply:
I don't think we should error on these. We know they should have been cutover, we can be smart enough to not break a production system. I could go either way on even logging it.